src/app/components_ngrx/planner-list/planner-list.component.ts

```diff
@@ -533,6 +533,13 @@ export class PlannerListComponent implements OnInit, OnDestroy, AfterViewChecked
     this.store.dispatch(new WorkItemActions.Reoder(payload));
   }
 
+  closePopover() {
```

Comment:
What exactly does this function do? Should have a comment for this one.

Reply:
It closes the popover on scroll. I will add the comment.